python/ecs/ecs-service-with-advanced-alb-config/app.py

```diff
@@ -23,7 +23,7 @@
 asg = autoscaling.AutoScalingGroup(
     stack, "DefaultAutoScalingGroup",
     instance_type=ec2.InstanceType.of(
-                         ec2.InstanceClass.STANDARD5,
```

Comment:
STANDARD5 translates to the m5 instance family... of which there is no micro size